src/modules/0.0.23/aws_vpc/entity/peering_connection.ts

```diff
@@ -0,0 +1,51 @@
+import { Column, Entity, JoinColumn, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
+
+import { cloudId } from '../../../../services/cloud-id';
+import { Vpc } from './vpc';
+
+export enum PeeringConnectionState {
+  ACTIVE = 'active',
+  EXPIRED = 'expired',
+  FAILED = 'failed',
+  INITIATING_REQUEST = 'initiating-request',
+  PENDING_ACCEPTANCE = 'pending-acceptance',
+  PROVISIONING = 'provisioning',
+  REJECTED = 'rejected',
+}
+
+@Entity()
+export class PeeringConnection {
```

Comment:
Are peerings region specific or global?

Reply:
They are region specific, but I thought keeping an extra `region` column would be redundant since `requester.region` and `accepter.region` for the VPCs are available. What's your idea?

Reply:
Actually the 2 VPCs **can** be from two different regions.

Reply:
oh, nevermind then. So the peering has to be in the requester one? or wherever?

Reply:
The peering creation should be on the `requester` region, but the accept call should be originating from the `accepter` region.